### PR TITLE
base.config: enable apparmor

### PR DESCRIPTION
**Pull request by juippis**
Bug: https://bugs.gentoo.org/834918

### CODE REVIEW

File: base.config [hunk 1]
```diff
@@ -62,3 +62,11 @@ CONFIG_KERNEL_GZIP=y
...
 ## Fix the path for Gentoo.
 CONFIG_MODPROBE_PATH="/sbin/modprobe"
+
+## Enable apparmor. Fedora only provides SELinux support.
+## https://bugs.gentoo.org/834918
+CONFIG_SECURITY_APPARMOR=y
+CONFIG_SECURITY_APPARMOR_HASH=y
+CONFIG_SECURITY_APPARMOR_HASH_DEFAULT=y
+# CONFIG_SECURITY_APPARMOR_DEBUG is not set
+# CONFIG_DEFAULT_SECURITY_APPARMOR is not set
```
Comment (mgorny) [inline base.config:72]: Do you need these two?

Comment (juippis) [reply base.config:72]: These only become visible after enabling `CONFIG_SECURITY_APPARMOR` but you should leave them disabled. I don't claim to understand how gentoo-kernel / gentoo-kernel-bin work but I saw other commented options in this file and assumed each option needs 0 or 1. But they are disabled by default so they can be left out if explicit "configure option" isn't required here.

Comment (mgorny) [reply base.config:72]: Well, I think either way is fine, as long as the end result is correct. I leave the final choice to you.

Comment (juippis) [reply base.config:72]: Personally I like it this way more, just to document all the flags that become available. But I can see it also bloating the file. 

I'd leave them here. `CONFIG_DEFAULT_SECURITY_APPARMOR` can be used to get apparmor loaded in boot without having to modify your grub.cfg.

**Pull request closed without merging**